Website/src/rendering/src/assets/css/layout/hero.css

```diff
@@ -1,7 +1,5 @@
 .section__hero {
   @apply m-0 bg-black bg-left bg-no-repeat bg-cover relative;
-
-  margin-top: 122px;
```

Comment:
At the bottom of this file, there is an alteration for Experience Editor:

```css
.experience-editor-active {
  .section__hero {
    @apply mt-0;
  }
}
```

This likely has to be moved and modified to affect the `<main>` element when in the Experience editor with the top menu that is not sticky.

There is logic in `Layout.tsx` regarding this EE support as well.

Reply:
Done.